excise/plugin.py

```diff
@@ -270,15 +261,19 @@ def calculate_checkout_line_total(
             return base_total
 
         checkout = checkout_line.checkout
+
         if not _validate_checkout(checkout, [checkout_line]):
             return base_total
 
         taxes_data = get_checkout_tax_data(checkout, discounts, self.config)
 
         if not taxes_data or "Error" in taxes_data["Status"]:
-            logger.debug("Error in tax response %s")
             return base_total
 
+        # store itemized tax information in Checkout metadata for optional display on the frontend
+        tax_meta = json.dumps(taxes_data["TransactionTaxes"])
```

Comment:
I don't think that this is a proper way for storing tax information in checkout metadata.
If you really need this on the storefront, then IMHO you should convert the response from ATE to your structure and store there only an important field for you. 
- now, you don't have control, what data is exposed by checkout.metadata.
- I assume that most of these fields will not be needed on the storefront and we should save only important fields for us.
- ATE is currently under development, which means that the response can be changed, even if currently doesn't expose any important data, there is a possibility that this will happen in the future.